node.go

```diff
@@ -190,7 +191,7 @@ func (n *Node) Title() string {
 	if n.root == n.path {
 		return filepath.Base(n.root)
```

Comment:
I think this also needs special treatment (not sure if even `removeOrderNumber()`).